hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx

```diff
@@ -81,6 +89,14 @@ class NavBar extends React.Component<INavBarProps> {
             <span>Disk Usage</span>
             <Link to='/DiskUsage'/>
           </Menu.Item>
+          {
+            this.state.isheatmapavailable ?
+              <Menu.Item key='/HeatMap'>
+                <Icon type='bar-chart' />
+                <span>HeatMap</span>
```

Comment:
shouldn't it be Heatmap? previously I saw it with this spelling.
cc @devmadhuu 
```suggestion
                <span>Heatmap</span>
```